errors.go

```diff
@@ -0,0 +1,19 @@
+package srvclient
+
+import "fmt"
+
+type notFoundErr struct {
```

Comment:
Yeah, I originally had that but since we're not exposing it, I've seen places where they do this instead.